types/mongoose/v4/index.d.ts

```diff
@@ -1,6 +1,9 @@
-// Type definitions for Mongoose 4.7.1
+// Type definitions for Mongoose 4.7.2
```

Comment:
For future references, the patch version is meaningless and is only still present because our lint rules are disabled.

Reply:
OK. Thanks for your review. :+1: